aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -2429,14 +2523,33 @@ public void updateUserAttributes(final Map<String, String> userAttributes,
      * @throws Exception
      */
     @WorkerThread
-    public List<UserCodeDeliveryDetails> updateUserAttributes(final Map<String, String> userAttributes) throws Exception {
+    public List<UserCodeDeliveryDetails> updateUserAttributes(
+            final Map<String, String> userAttributes) throws Exception {
+        return updateUserAttributes(userAttributes, Collections.<String, String>emptyMap());
+    }
+
+    /**
+     * Sends a map of user attributes to update. If an attribute needs to
+     * be verified, then the verification delivery information is returned.
+     * @param userAttributes the attributes i.e. email
+     * @param clientMetadata A map of custom key-value pairs that is passed to the lambda function for
+     *                       custom workflow.
+     * @return verification delivery information
+     * @throws Exception
```

Comment:
description. Is there also any way we can specify what type of `Exception` specifically will be thrown by this function?

Reply:
This was already there so didn't update old documentation